src/components/User/Operations/OperationDetails.vue

```diff
@@ -12,12 +14,63 @@
 <script>
 import DetailsReader from '../../common/details/DetailsReader'
```

Comment:
Alias pls. For the future: if you work with some file, check that aliases used and fix it if not.

Reply:
do you mean the component DetailsReader is not present in the project and even such file is not existed in our file system?